event_types/event_types.py

```diff
@@ -488,9 +489,46 @@ def extract_df_from_dl2(root_filename):
         data_dict["me_tgrad_x"].extend(tuple(me_tgrad_x))
         data_dict["std_tgrad_x"].extend(tuple(std_tgrad_x))
 
+        # change NaNs or Infs in any of the variables to 0 and count them
+        nan_count = 0
```

Comment:
Since the for loop is over the iteration of uproot, is it an issue that you set the `nan_count` to zero here? You might be counting only the last step size this way, no? 
Or perhaps you don't mind since you just print out the warning for each step?

Reply:
You're right, this should have one less tab. I'm updating it. Thanks!